phidgets_steppers/README.md

```diff
@@ -0,0 +1,48 @@
+Phidgets stepper ROS 2 driver
+=============================
+
+This is the ROS 2 driver for Phidgets stepper.  The various topics, services, and parameters that the node operates with are listed below.
+
+Topics
```

Comment:
Maybe worth renaming this to "Publishers".

Reply:
So instead of Topics you think we should have two headings, Publishers and Subscribers?

Reply:
Yes - it's pretty important to distinguish whether the node publishes a topic or subscribes to it. The ROS wiki documentation macros call the headings "Published Topics" and "Subscribed Topics", and I like that naming even better.

By the way, I just noticed the other phidgets READMEs also only call the heading "Topics". That should be changed too, I guess (but that's outside the scope of *this* PR).

Reply:
Makes sense, I will change it. Do you have a link to a particular README that we can use as a template?

Reply:
I think the rest of the README is fine, and it's consistent with the READMEs of the other packages here, so I would simply change the heading to "Published Topics" and be done with it. :)

Reply:
> I think the rest of the README is fine, and it's consistent with the READMEs of the other packages here, so I would simply change the heading to "Published Topics" and be done with it. :)

Oh yeah, I totally agree with @mintar that this should be "Published Topics" and "Subscribed Topics".  I just forgot to do that when I did the original documentation.  I'll open a PR soon here to change the rest of the README's in this repository to have those titles.